course/frontend/array-shuffle/SpecRunner.html

```diff
@@ -1,24 +1,22 @@
 <!DOCTYPE html>
```

Comment:
UPDATE this branch with upstream/develop, RESET it if necessary, and then apply your changes without affecting the existing code ,)

Reply:
I think so It's done.